### ISSUE TITLE
Context Menu on Paths

### ISSUE THREAD

**Issue by winsandymyint** [aratcliffe/Leaflet.contextmenu#60]
Could you please let me know that we can use the context menu on paths? 
It is working on 'map', but not ok with 'paths'. I used angular-leaflet-directive. Thanks

**Comment by aratcliffe** [comment]
Yes I have used it with paths. I'm not an angular user but can't see why it would affect the use of the plugin for paths. There's a basic angular example that I included in the examples directory but that does not add menu items to any map layers. You could try creating a simple test case that doesn't use angular and if you're still seeing the problem there I can look into it for you.
